jsonnetlib/k8s.jsonnet

```diff
@@ -83,16 +83,12 @@ cfg {
     memoryRequest=root.cronMemoryRequest,
     memoryLimit=root.cronMemoryLimit,
     backoffLimit=0,
-    ttl=null,
+    ttl=3600,
```

Comment:
@dzvon please help extract the value into a variable in `config.json` to align with jobTTL ([ref](https://github.com/theplant/plantbuild/blob/2334b7f1d6ff8c98bd2b66f0b126775c2ec75fdb/jsonnetlib/config.jsonnet#L30))

Reply:
Added a new variable `cronjobTTL` for this.